environments/icds-cas/postgresql.yml

```diff
@@ -162,3 +161,14 @@ dbs:
       create: False
       django_migrate: False
       hq_acceptable_standby_delay: 0
+  logical:
+    - name: test_pglogical_low
+      host: testdeleteshardlow
+      master_host: testdeleteshard
+      master_db_name: commcarehq_p10_test_delete
+      replication_set: [927, 975]
```

Comment:
This configuration means "replicate shards 927 - 975 inclusive from database `commcarehq_p10_test_delete` on machine `testdeleteshard` to database `test_pglogical_low` on machine `testdeleteshardlow`

Reply:
I don't disagree. I think I was using this as a quick way to create the databases during setup. Will think a bit more

Reply:
Finally did this https://github.com/dimagi/commcare-cloud/pull/3080/files/c282d261c8e1d42278919127fe7acaebf4c59a71..790fb08085a1c4a5e3ad7be9dc8b66a4f4719766